torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1330,18 +1330,9 @@ void ProcessGroupNCCL::abortCommsFromMap(
   for (auto& it : ncclCommsMap) {
     auto& devName = it.first;
     auto& ncclComm = it.second;
-    at::cuda::OptionalCUDAGuard gpuGuard;
-    at::DeviceIndex deviceIndex = getIndexFromDeviceKey(devName);
```

Comment:
Shall we also remove `getIndexFromDeviceKey`?

Reply:
Oh do you mean this function has no use now? If so, yeah, we should remove it